src/containers/ArchiveGroupButtonContainer/ArchiveGroupButtonContainer.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ImmutablePropTypes from 'react-immutable-proptypes';
+import { connect } from 'react-redux';
+
+import ArchiveGroupButton from '../../components/buttons/ArchiveGroupButton';
+import { setArchived } from '../../redux/modules/groups';
+import {
+  groupSelector,
+  groupArchivedPendingChange
+} from '../../redux/selectors/groups';
+import ResourceRenderer from '../../components/helpers/ResourceRenderer';
+
+const ArchiveGroupButtonContainer = ({
+  group,
+  pending,
+  setArchived,
+  bsSize = undefined
+}) =>
+  <ResourceRenderer resource={group}>
+    {({ directlyArchived, permissionHints }) =>
+      <ArchiveGroupButton
+        archived={directlyArchived}
+        pending={pending}
+        setArchived={setArchived}
+        disabled={!permissionHints.update}
```

Comment:
I think that users without update permission should not see the button at all (they should not see a disabled button).